src/components/Contacts.js

```diff
@@ -0,0 +1,114 @@
+// Contacts.js
+import React, { useState } from "react";
+import contacts from "./../contacts.json";
+
+const Contacts = () => {
+  const allContacts = contacts;
+  const fiveCont = allContacts.slice(0, 5);
+
+  const [sumContact, setSumContact] = useState(fiveCont);
+
+  /*  console.log(fiveCont)
+    console.log(allContacts) */
+
+  const randomContact = (contact) => {
+    const obtainRandomContact =
+      contact[Math.floor(Math.random() * contact.length)];
+
+    return setSumContact([...sumContact, obtainRandomContact]);
+  };
+
+  const sortName = (contact) => {
+    const sortedNames = contact.sort((x, y) => {
+      if (x.name > y.name) {
+        return 1;
+      } else if (x.name < y.name) {
+        return -1;
+      } else {
+        return 0;
+      }
+    });
+
+    return setSumContact([...sumContact, sortedNames]);
+  };
+
+  const sortPopularity = (contact) => {
+    const sortedPop = contact.sort((x, y) => {
+      return y.popularity - x.popularity;
+    });
+
+    return setSumContact([...sumContact, sortedPop]);
```

Comment:
Hay que pasar el sortedPop de la siguente manera ya que si lo pasas asi, el siguente elemento de tu arreglo sera otro arreglo con toda la data ordenada por eso te salen cards sin foto por que en realidad son arreglos 👀 
```suggestion
    return setSumContact([...sortedPop]);
```